slosh/benches/all.rs

```diff
@@ -92,7 +92,8 @@ fn run_float_script(n: usize, m: f32, expected: f64) {
     let last = run_reader(&mut reader);
     match last {
         Ok(Value::Float(f)) => {
-            assert_eq!(f64::from(f), expected);
+            assert_eq!(f.to_string(), expected.to_string());
```

Comment:
remove this line and use the original. Since this is a benchmark we don't need to go through the extra cycles to check the string equality. You already made some good functions to do that.

Plus, we're keeping track of bench runs. And runs are kept track of and compared by name. So if we we change one it's important to rename it so you don't mix up benchmarks. Since the regular assert_eq works, and because this was more of a formatting issue when dealing with printed values in slosh (hence the fixes to bench.slosh) I think it makes sense to leave it as is. And I don't see any merit in making a new test that benchmarks some math in addition to the new to_string command. If you're representing a number as a string, there's going to be an alloc penalty anyway not to mention the fact that you're probably printing which is very expensive in terms of time. We probably should add some benchmarks for some typical actions of that sort, but we don't want those in these existing benchmark functions because I think it's a separate concern.

we do need to get bench.slosh running in CI now though.